shell/run_shell_cmd.go

```diff
@@ -51,13 +65,21 @@ func RunShellCommand(terragruntOptions *options.TerragruntOptions, command strin
 		// bad path, binary not executable, &c
 		return errors.WithStackTrace(err)
 	}
+
+	output, err := readStdoutAndStderr(stdoutIn, stderrIn, terragruntOptions, outToErr)
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+
 	cmdChannel := make(chan error)
 	signalChannel := NewSignalsForwarder(forwardSignals, cmd, terragruntOptions.Logger, cmdChannel)
 	defer signalChannel.Close()
 
-	err := cmd.Wait()
+	err = cmd.Wait()
 	cmdChannel <- err
 
+	terragruntOptions.Logger.Println(fmt.Sprintf("And the output is: %v", output))
```

Comment:
Don't forget to remove this later (I assume it's just for debugging?).

Reply:
Yes, this is for debugging at this point. Didn't want to change the return values just yet but wanted to double check that I do capture the output.